upi/openstack/security-groups.yaml

```diff
@@ -70,6 +70,17 @@
       port_range_min: 6443
       port_range_max: 6443
 
+  - name: 'Create master-sg IPv6 rule "OpenShift API"'
+    openstack.cloud.security_group_rule:
+      security_group: "{{ os_sg_master }}"
+      ether_type: IPv6
+      protocol: tcp
+      port_range_min: 6443
+      port_range_max: 6443
+    when:
+    - os_subnet_range_v6 is defined
+    - "{{ (os_subnet_range|ansible.utils.ipv4) and (os_subnet6_range|ansible.utils.ipv6) }}"
```

Comment:
Let's not bother about `os_subnet_range` in the IPv6 rules, as the two are unrelated. Let's just check that os_subnet_range_v6 is defined and is a proper IPv6 address.